calendar.js

```diff
@@ -0,0 +1,60 @@
+const request = require('request');
+const cheerio = require('cheerio');
+const baseUrl = 'https://k2nblog.com/';
+
+const dateObj = new Date();
+const month = dateObj.getMonth() + 1; 
+const day = dateObj.getDate(); 
+const year = dateObj.getFullYear();
+
+function _getCurrentUrl() {
+    return `${baseUrl}/${year}/${month}/${day}/`;
+}
+
+function _asyncRequest(url) {
+    return new Promise( (resolve, reject) => {
+        request(url, (error, res, body) => {
+            if (!error && (res.statusCode == 200 || res.statusCode == 301)) {
+                resolve(body);
+            } else {
+                reject(error);
+            }
+        });
+    });
+}
+
+function getCalendarDataSource() {
+    return _getCurrentUrl();
+};
+
+async function getCalendarData() {
+    const allNewReleases = {}
+    const releaseTypeRegex = /\[([^)]+)\][ \t]+/;
+    const fileTagRegex = /[ \t]+\([^\][]*\)$/;
+    let currentPageNum = 1;
+
+    while(1) {
```

Comment:
Is this the right way to do this?

Reply:
Whoops missed this. See https://github.com/JustYourPythonEnvironment/JokBot/pull/2#discussion_r224338365